eng/common/scripts/stress-testing/stress-test-deployment-lib.ps1

```diff
@@ -156,11 +150,24 @@ function DeployStressPackage(
     }
     $imageTag += "/$($pkg.Namespace)/$($pkg.ReleaseName):${deployId}"
 
-    $dockerFilePath = "$($pkg.Directory)/Dockerfile"
+    $dockerFilePath = if ($pkg.Dockerfile) {
+        [System.Io.Path]::Combine($pkg.Directory, $pkg.Dockerfile).ToString()
```

Comment:
I would stick to using one method of joining paths, unless there's something I'm missing in the difference of usage here?

```suggestion
        Join-Path $pkg.Directory $pkg.Dockerfile
```